plugins/inventory/icinga2.py

```diff
@@ -63,6 +65,20 @@
 host_filter: \"linux-servers\" in host.groups
 validate_certs: false
 inventory_attr: name
+groups:
+  # simple name matching
+  webservers: inventory_hostname.startswith('web')
+
+  # using icinga2 template
+  databaseservers: "'db-template' in (icinga2.templates|list)"
```

Comment:
Shouldn't this also be `icinga2_attributes`?
```suggestion
  databaseservers: "'db-template' in (icinga2_attributes.templates|list)"
```

Reply:
Yes, I missed that. Thanks!